search/intermediate_result.cpp

```diff
@@ -24,7 +24,6 @@ namespace search
 {
 
 /// All constants in meters.
```

Comment:
my note was not about comments style - like a group or for each const. it was about counting - comment says about many constants, while there is only one constant.
it is not critical. up to you.